packages/gitbook/src/app/(space)/~gitbook/pdf/pointer.ts

```diff
@@ -0,0 +1,15 @@
+import { SiteContentPointer, SpaceContentPointer } from '@/lib/api';
+import { getSiteContentPointer, getSpacePointer } from '@/lib/pointer';
+
+/**
+ * PDF generation can be done at the site level (e.g. docs.foo.com/~gitbook/pdf) or
+ * at the space level (e.g. open.gitbook.com/~space/:spaceId/~gitbook/pdf). This function
```

Comment:
Let's just add a small explanation that this is for in-app PDF export of private content